src/plugins/env.ts

```diff
@@ -3,14 +3,29 @@ import fastifyEnv from "@fastify/env";
 
 const schema = {
   type: "object",
-  required: ["PROXY_ORIGIN", "PROXY_UPSTREAM"],
+  required: [
+    "PROXY_ORIGIN",
```

Comment:
for another PR (since we never reivied official code. 

But if this codebase will contain multiple proxies, you better specify what is this proxy sth like UNISWAP_TOKENS_PROXY_ORIGIN ?

Reply:
Definitely, happy to follow up on that one.

Reply:
`TOKENS_PROXY_ORIGIN` instead?
This is not uniswap specific